lib/vagrant-service-manager/plugin_util.rb

```diff
@@ -111,6 +111,7 @@ def self.execute_and_exit_on_fail(machine, ui, command)
 
       def self.execute_once(machine, ui, command)
         machine.communicate.sudo(command) do |_, data|
+          next if data.chomp.empty?
```

Comment:
@hferentschik , to give more insights here, when we run command with `machine.communicate.sudo(command)` and use block format as done here, we basically get 
stream type( `_` ignored here) and data executed by that command. More info [here](https://github.com/mitchellh/vagrant/blob/master/lib/vagrant/plugin/v2/communicator.rb#L94-L102).

Now, in case of Windows OS, the `data` part contains expected info on 2nd iteration which was not the case in 1.8.1 (didn't check later versions). In first iteration, `data` part is empty and hence `next`.
Hope it make sense.